.eslintrc

```diff
@@ -9,6 +9,17 @@
         "before": "readonly",
         "beforeEach": "readonly"
       }
+    },
+    {
+      "files": ["**/*.js"],
```

Comment:
Isn’t this the default?

Reply:
I'm getting Missing property "files".

Reply:
Now I could probably remove this warning too 😄